backend/scripts/gcp/marketplace/startup_script.sh

```diff
@@ -0,0 +1,182 @@
+echo "DSHOP setup beginning!"
+
+cat > ~/setup_script.sh << 'FULL_SCRIPT_DELIMITER'
+set -e
+set -x
+
+CODENAME=`grep -Po 'VERSION="[0-9]+ \(\K[^)]+' /etc/os-release`
+
+#################
+# Server packages
+#################
+curl -sL https://deb.nodesource.com/setup_10.x | bash -
+apt update
+apt upgrade -y
+apt-get install -y nodejs build-essential git wget gnupg ca-certificates
+npm install -g yarn pm2
+
+###################
+# Openresty install
+###################
+wget -O - https://openresty.org/package/pubkey.gpg | sudo apt-key add -
+echo "deb http://openresty.org/package/debian $CODENAME openresty" \
+    | sudo tee /etc/apt/sources.list.d/openresty.list
+apt-get update
+apt-get -y install openresty luarocks
+luarocks install lua-resty-auto-ssl
+mkdir /etc/resty-auto-ssl
+chown www-data:www-data /etc/resty-auto-ssl
+openssl req -new -newkey rsa:2048 -days 3650 -nodes -x509 \
+    -subj '/CN=sni-support-required-for-valid-ssl' \
+    -keyout /etc/ssl/resty-auto-ssl-fallback.key \
+    -out /etc/ssl/resty-auto-ssl-fallback.crt
+
+###############
+# Install dshop
+###############
+BACKEND_PATH="/app/dshop/backend"
```

Comment:
Lacks `export` this will be undefined when used later.  Perhaps skip using vars here since it's only used once.

Reply:
Ah yes. Thank you, great catch!

Reply:
Actually, you only need to export for child processes to get the env var.
In this case the var is only used in the script, so unless I'm missing something there is no need to export?

Reply:
:shrug: it was really early.  Still looks unnecessary to me.